src/main/java/com/algorand/algosdk/transaction/MethodCallParams.java

```diff
@@ -35,7 +36,7 @@ public class MethodCallParams {
     public final List<Address> foreignAccounts;
     public final List<Long> foreignAssets;
     public final List<Long> foreignApps;
-    public final List<AppBoxReference> boxReferences;
+    public List<AppBoxReference> boxReferences;
```

Comment:
I believe this field is meant to stay `final`?

Reply:
@jasonpaulos Thanks for catching and sorry for missing on my end.  It's addressed in https://github.com/algorand/java-algorand-sdk/pull/408/commits/d79e9fcd7f25bd82fbb6ef53fab0fadca7faaaf3.  Can you resolve if it looks good?